### PR TITLE
Upgraded to Angular 13

### PR DESCRIPTION
**Pull request by alansemenov**

### CODE REVIEW

**Comment by alansemenov** [comment]
No longer relevant after Dependabot upgrades

**Comment by reisfmb** [comment]
Fix was based on:
https://github.com/angular/angular/issues/44026#issuecomment-958385983
https://github.com/angular/angular/blob/master/integration/platform-server/base-config.mjs#L3

---

**FYI**

Here are the reasons for module changes from cjs to mjs: 

`webpack.common.js` imports now a mjs: `'@angular/compiler-cli/linker/babel'`, therefore it is now a mjs, and would be good that all files that depends on  `webpack.common.js`  also changes to mjs, they are:
`webpack.aot.js`
`webpack.dev.js`
`webpack.prod.js` 

First one is already converted since it is the one executed by the gradle build.

If this is accepted as the correct solution to the problem, then it will be also necessary to update `webpack.dev.js` and `webpack.prod.js` to mjs.

File: package.json [hunk 1]
```diff
@@ -8,43 +8,44 @@
...
+    "chart.js": "^3.4.0",
     "chartjs-plugin-annotation": "0.5.7",
     "core-js": "^3.21.1",
     "crypto-js": "4.1.1",
     "hammerjs": "^2.0.8",
     "jquery": "^3.5.0",
     "material-design-icons": "3.0.1",
     "materialize-css": "1.0.0",
-    "ng2-charts": "^2.3.0",
+    "ng2-charts": "^3.0.8",
```
Comment (poi33) [inline package.json:30]: breaking version anything need updating?

**Review by poi33 — APPROVED** [review]

**Comment by alansemenov** [comment]
@reisfmb Seems like a fine solution, please fix other files. Also note conflicts in this PR that should also be solved.

File: .github/workflows/enonic-gradle.yml [hunk 2]
```diff
@@ -3,25 +3,123 @@ name: Gradle Build
 on: [ push ]
```
Comment (poi33) [inline .github/workflows/enonic-gradle.yml:3]: Why was this file changed? 
This is not part of the angular change.

Comment (poi33) [reply .github/workflows/enonic-gradle.yml:3]: This is also not an up to date build and deploy action.

File: gradle.properties [hunk 3]
```diff
@@ -5,6 +5,4 @@ group = com.enonic.app
 projectName = officeleague
 appName = com.enonic.app.officeleague
 xpVersion = 7.3.0
-
-version = 3.1.1
-version=3.1.2-SNAPSHOT
+version = 3.1.1-SNAPSHOT
```
Comment (poi33) [inline gradle.properties:8]: 3.1.1 is already released. 

This would be 3.1.2-SNAPSHOT

File: src/main/resources/lib/pwa/manifest-controller.js [hunk 4]
```diff
@@ -2,15 +2,14 @@ var portalLib = require('/lib/xp/portal');
 var mustache = require('/lib/mustache');
 
 exports.get = function(req) {
-
-    const startUrl = "./";
+    var startUrl = portalLib.url({ path: "/", type: "absolute" });
```
Comment (poi33) [inline src/main/resources/lib/pwa/manifest-controller.js:5]: This stopped working at some point. The new (server) uri is: `webapp/com.enonic.app.officeleague/`

**Review by poi33 — CHANGES_REQUESTED** [review]

File: src/main/resources/lib/pwa/manifest-controller.js (hunk 4 above)
Comment (alansemenov) [reply src/main/resources/lib/pwa/manifest-controller.js:5]: @reisfmb let's keep it as it was. this is tested and works.

File: .github/workflows/enonic-gradle.yml (hunk 2 above)
Comment (reisfmb) [reply .github/workflows/enonic-gradle.yml]: My bad. I merged the master intro this branch in order to solve the conflict. Now I've done it properly. @poi33 please check last commit.

**Review by poi33 — APPROVED** [review]
